e2e/client/playwright/custom-filed.spec.ts

```diff
@@ -0,0 +1,120 @@
+import {test, expect} from '@playwright/test';
+import {Monitoring} from './page-object-models/monitoring';
+import {Settings} from './page-object-models/settings';
+import {restoreDatabaseSnapshot, s} from './utils';
+
+async function checkVisibilityInAuthoring(page, field) {
+    const monitoring = new Monitoring(page);
+
+    await page.goto('/#/workspace/monitoring');
+    await monitoring.selectDeskOrWorkspace('Sports');
+    await monitoring.createArticleFromTemplate('story');
+    await expect(page.locator(s('authoring', `authoring-field=${field}`))).toBeVisible();
+}
+
+async function addFieldToContentProfile(page, place, field) {
+    const settings = new Settings(page);
+
+    await page.goto('/#/settings/content-profiles');
+    await settings.addFieldInContentProfile('Story', place, field);
+    await expect(
+        page.locator(s('content-profile-editing-modal', `content-profile-item=${field}`)),
+    ).toBeVisible();
+    await page.locator(s('content-profile-editing-modal')).getByRole('button', {name: 'Save'}).click();
+}
+
+test('creating a custom text field', async ({page}) => {
+    await restoreDatabaseSnapshot();
+    await page.goto('/#/settings/vocabularies');
+
+    await page.locator(s('metadata-tabs')).getByRole('button', {name: 'Custom text fields'}).click();
+    await page.locator(s('metadata-content')).getByRole('button', {name: 'Add new'}).click();
+    await page.locator(s('vocabulary-modal')).getByLabel('id').fill('textfield');
+    await page.locator(s('vocabulary-modal')).getByLabel('name').fill('custom text field 2');
+    await page.locator(s('vocabulary-modal')).getByRole('button', {name: 'Save'}).click();
+    await expect(page.locator(s('metadata-content', 'vocabulary-item=custom text field 2'))).toBeVisible();
+    await addFieldToContentProfile(page, 'Content', 'custom text field 2');
+    await checkVisibilityInAuthoring(page, 'custom text field 2');
+});
+
+test('creating a custom date field', async ({page}) => {
+    await restoreDatabaseSnapshot();
+    await page.goto('/#/settings/vocabularies');
+
+    await page.locator(s('metadata-tabs')).getByRole('button', {name: 'Custom date fields'}).click();
+    await page.locator(s('metadata-content')).getByRole('button', {name: 'Add new'}).click();
+    await page.locator(s('vocabulary-modal')).getByLabel('id').fill('datefield');
```

Comment:
When writing tests, avoid using strings for input that look like variable/function names. `custom date field 2` is a good one, so then for ID use `custom-date-field-2`. Also adjust this in other tests in this file.